tests/flakes.sh

```diff
@@ -722,3 +723,8 @@ git -C $flakeB commit -a -m 'Foo'
 
 # Test list-inputs with circular dependencies
 nix flake metadata $flakeA
+
+# Test flake in store does not evaluate
```

Comment:
The comment probably isn’t really clear out-of-context. Something like below might be clearer

```suggestion
# A store path containing a `flake.nix` should be treated as a store path, and not a flake
```

Also it might be worth adding a test to ensure that for example `file:/nix/store/foobar` will be treated as a flake url and not a store path